.github/workflows/schema_spy.yml

```diff
@@ -0,0 +1,138 @@
+          password: ${{ secrets.DOCKER_TOKEN }}
+
+      # Run the shell commands using the AWS environment variables
+      - name: "Build & Publish Database Documentation"
+        env:
+          AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        run: |
+          export BRANCH_NAME=${GITHUB_REF##*/};
+          echo "SHA: ${GITHUB_SHA}";
+          echo "ACTION/BRANCH_NAME: ${BRANCH_NAME}";
+          echo "GR: ${GITHUB_REF}";
+          echo "PWD: $(pwd)";
+
+          echo "POSTGRES_PASSWORD=visionzero" > .env;
+          echo "POSTGRES_USER=visionzero" >> .env;
+          echo "POSTGRES_DB=atd_vz_data" >> .env;
+          echo "POSTGRES_HOST_AUTH_METHOD=trust" >> .env;
+
+          docker compose -f docker-compose-github-schema-spy-action.yml up -d postgis;
+          sleep 10;
+          docker compose -f docker-compose-github-schema-spy-action.yml up -d graphql-engine;
+          sleep 10;
+          cd database;
+          hasura --skip-update-check --database-name=default migrate apply;
+          cd ..;
+          sleep 10;
+          chmod a+rwX database/schemaspy_docs;
+          docker compose -f docker-compose-github-schema-spy-action.yml run schemaspy --debug;
+
+      - name: "Determine S3 Destination Directory"
+        id: s3-dest-dir
+        run: |
+          export BRANCH_NAME=${GITHUB_REF##*/};
+          if [ "$BRANCH_NAME" = "production" ]; then
+            echo "::set-output name=dest_dir::vision-zero-production";
+          else
+            echo "::set-output name=dest_dir::vision-zero-staging";
+          fi
+
+      - uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --delete
+        env:
+          AWS_S3_BUCKET: db-docs.austinmobility.io
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          SOURCE_DIR: 'database/schemaspy_docs'
+          DEST_DIR: ${{ steps.s3-dest-dir.outputs.dest_dir }}
+
+      # - name: "Invalidate CloudFront"
+      #   env:
+      #     AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+      #     AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      #     AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      #   run: |
+      #     export BRANCH_NAME=${GITHUB_REF##*/};
+      #     echo "SHA: ${GITHUB_SHA}";
+      #     echo "ACTION/BRANCH_NAME: ${BRANCH_NAME}";
+      #     echo "GR: ${GITHUB_REF}";
+      #     echo "PWD: $(pwd)";
+          
+      #     if [ "$BRANCH_NAME" = "production" ]; then
+      #       INVALIDATION_PATH="/vision-zero-production/*";
+      #     else
+      #       INVALIDATION_PATH="/vision-zero-staging/*";
+      #     fi
+
+      #     aws cloudfront create-invalidation --distribution-id E1QMRCYLR76UOL --paths "$INVALIDATION_PATH"
```

Comment:
This will come back in if we decide to keep schema spy after we identify and enhance the IAM account used in the GH secrets.